config/locales/views/privacy/de.yml

```
@@ -0,0 +1,39 @@
+    understand_users: "Um besser zu verstehen, wie Benutzer auf die CircuitVerse-Plattform zugreifen und sie nutzen, sowohl auf aggregierter als auch auf individueller Basis, und für andere Forschungs- und Analysezwecke."
+    evaluate_improve: "Um die CircuitVerse-Plattform zu evaluieren und zu verbessern und neue Produkte und Dienstleistungen zu entwickeln."
+    comply_legal: "Um gesetzlichen Verpflichtungen nachzukommen, als Teil unseres allgemeinen Geschäftsbetriebs und für andere geschäftliche Verwaltungszwecke."
+    investigate: "Soweit wir es für erforderlich halten, illegale Aktivitäten, Betrugsverdacht oder potenzielle Bedrohungen für die Sicherheit einer Person oder Verstöße gegen unsere Nutzungsbedingungen oder diese Datenschutzrichtlinie zu untersuchen, zu verhindern oder Maßnahmen zu ergreifen."
+    content_activity_heading: "Ihre Inhalte und Aktivitäten"
+    content_activity_description: "Ihre Inhalte, einschließlich Ihres Namens, Profilbilds, Profilinformationen und bestimmter zugehöriger Aktivitätsinformationen, sind für andere Benutzer der CircuitVerse-Plattform verfügbar und können öffentlich eingesehen werden. Die öffentliche Ansicht umfasst die Verfügbarkeit für nicht registrierte Besucher und kann auftreten, wenn Benutzer Ihre Inhalte über andere Websites oder Dienste teilen. Darüber hinaus können Ihre Inhalte von Suchmaschinen indiziert werden."
+    anonymized_heading: "Aggregierte und anonymisierte Daten"
+    anonymized_description: "Wir können aggregierte oder de-identifizierte Informationen mit Dritten für Forschungs-, Marketing-, Analyse- und andere Zwecke teilen, vorausgesetzt, solche Informationen identifizieren keine bestimmte Person."
+    tracking_heading: "Tracking"
+    analytics_tools_heading: "Analysetools"
+    analytics_tools_description_html: >-
+      Wir können interne und externe Analysetools verwenden, einschließlich <a 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)